lib/services/internalEngine/bootstrap.js

```diff
@@ -65,9 +85,8 @@ class InternalEngineBootstrap {
       }
     })
       .then(() => {
-        const promises = ['anonymous', 'default', 'admin'].map(roleId => {
-          return this.db.createOrReplace('roles', roleId, this.kuzzle.config.security.default.role);
-        });
+        const promises = ['anonymous', 'default', 'admin']
+          .map(roleId => this.db.createOrReplace('roles', roleId, this.kuzzle.config.security.default.role));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 109 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)